docs/resources/ocean_aks_np.md

```diff
@@ -306,4 +306,49 @@ update_policy {
     node_names = ["aks-omnp123456-7890-vmss000001"]
   }
 }
+```
+<a id="scheduled-task"></a>
+## Scheduled Task
+* `scheduled_task` - (Optional) An object used to specify times when the cluster will turn off. Once the shutdown time will be over, the cluster will return to its previous state.
+  * `shutdown_hours` - (Optional) An object used to specify times that the nodes in the cluster will be taken down.
+    * `is_enabled` - (Optional) Flag to enable or disable the shutdown hours mechanism. When False, the mechanism is deactivated, and the cluster remains in its current state.
+    * `time_windows` - (Required) The times that the shutdown hours will apply. Required if isEnabled is true.
```

Comment:
re-check this, as per document it is optional attribute

Reply:
Required if isEnabled is true.